spec/dummy/db/schema.rb

```diff
@@ -11,32 +11,39 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20140610093243) do
+ActiveRecord::Schema.define(version: 20150617152432) do
 
-  create_table "document_numbers", force: true do |t|
+  create_table "document_numbers", force: :cascade do |t|
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.